app/mutations/application/updateApplicationStatusByRowIdMutation.ts

```diff
@@ -0,0 +1,32 @@
+import {graphql} from 'react-relay';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
+import {
+  updateApplicationStatusByRowIdMutation as updateApplicationStatusByRowIdMutationType,
+  updateApplicationStatusByRowIdMutationVariables
+} from 'updateApplicationStatusByRowIdMutation.graphql';
+import BaseMutation from '../BaseMutation';
+
+const mutation = graphql`
+  mutation updateApplicationStatusByRowIdMutation(
+    $input: UpdateApplicationStatusByRowIdInput!
+  ) {
+    updateApplicationStatusByRowId(input: $input) {
+      applicationStatus {
```

Comment:
Add clientMutationId to the mutation
```suggestion
     clientMutationId
      applicationStatus {
```

Reply:
Added